src/TRE-API/Controllers/SubmissionController.cs

```diff
@@ -352,5 +352,103 @@ public IActionResult SimulateSubmissionProcessing(Submission sub)
                 throw;
             }
         }
+
+        #region ForTest
+
+        /// <summary>
+        /// create secret for minio
+        /// </summary>
+        /// <param name="accessKey">Access Key identifier</param>
+        /// <param name="secretKey">Secret Key identifier</param>
+        [HttpPost("create-minio-secret/{accessKey}/{secretKey}")]
```

Comment:
Should this be on SubmissionController?

it's not directly about handling a Submission, and I'm not sure I fully know the context for the functionality.

The end point is very RPC. It's an action `create-minio-secret` whereas really a Minio secret should be a resource, and a route to create one would be something like `POST /api/v1/minio-secret`

But also it shouldn't have `minio` in the route because that's an implementation detail, and we intend this to be a standard API (unless this isn't part of the standard)

So maybe `storage-secret` or something? Though that's too general and unhelpful. Like I say, I don't really understand what this is for.